pkg/capture/provider/network_capture_unix.go

```diff
@@ -38,8 +40,8 @@ func NewNetworkCaptureProvider(logger *log.ZapLogger) NetworkCaptureProviderInte
 	}
 }
 
-func (ncp *NetworkCaptureProvider) Setup(captureName, nodeHostname string) (string, error) {
-	captureFolderDir, err := ncp.NetworkCaptureProviderCommon.Setup(captureName, nodeHostname)
+func (ncp *NetworkCaptureProvider) Setup(captureName, nodeHostname string, startTimestamp time.Time) (string, error) {
+	captureFolderDir, err := ncp.NetworkCaptureProviderCommon.Setup(captureName, nodeHostname, startTimestamp)
```

Comment:
Following from the comment on the prev file: Instead of passing timestamp down the `Setup` call chain, we can use ncp.StartTimestamp, and similarly for other objects implementing this interface? I am assuming it is going to be constant for the lifecycle of capture